tests/backend/test_tf_import.py

```diff
@@ -71,24 +71,24 @@ class TestFrontendImport:
 
     def test_incorrect_tf_version(self, monkeypatch):
         """Test that an exception is raised if the version
-        of TensorFlow installed is not version 1.3"""
+        of TensorFlow installed is version 2.x"""
```

Comment:
docstring is not correct

Reply:
docstring fixed.
Regarding the wigner function, if it does fall back to a numpy implementation then it's not trainable indeed.